docker-compose.yml

```diff
@@ -28,6 +29,7 @@ services:
       - ./dev/example.sql.xz:/example.sql.xz:z
       - ./dev/db/post-migrations.sql:/post-migrations.sql:z
       - ./bin/wait-for-db:/usr/local/bin/wait-for-db:z
+      - postgres_data:/var/lib/postgresql/data/
```

Comment:
Should this be part of this PR?

Reply:
I've updated the PR title and description to reflect the two commits